dist/IO/lib/IO/Handle.pm

```diff
@@ -368,7 +372,7 @@ sub fdopen {
 
     if (ref($fd) && "".$fd =~ /GLOB\(/o) {
```

Comment:
"".$fd doesn't do what you think it does..?

=~has higher precedence; should have used parens